tests/sentry/rules/conditions/test_event_attribute.py

```diff
@@ -856,3 +856,15 @@ def test_attr_is_in(self):
             data={"match": MatchType.IS_IN, "attribute": "platform", "value": "python"}
         )
         self.assertDoesNotPass(rule, event)
+
+    def test_attr_not_in(self):
+        event = self.get_event()
+        rule = self.get_rule(
+            data={"match": MatchType.NOT_IN, "attribute": "platform", "value": "php, python"}
```

Comment:
thoughts on adding some test cases around different `values` here? thinking like empty string, strings with spaces / w/o etc -- i'd just do it with like a pymark.paramterize too

Reply:
I don't see that as particularly useful atm.

since it's a text-input, the values always going to be a string.
the different values will all just be unmatching strings 🤷

Reply:
the value add is really in resiliency in the code and to reduce number of issues we see in production. if there are additional test cases we can reduce the number of issues while you still have strong context over the feature.